src/main/kotlin/com/sovdee/skriptbounds/elements/properties/ExprDimensions.kt

```diff
@@ -27,20 +36,85 @@ class ExprDimensions : SimplePropertyExpression<AxisAlignedBox, Double>() {
         return super.init(expressions, matchedPattern, isDelayed, parseResult)
     }
 
-    override fun convert(from: AxisAlignedBox): Double {
-        return when (dimension) {
-            Dimension.LENGTH -> from.length
-            Dimension.WIDTH -> from.width
-            Dimension.HEIGHT -> from.height
+    override fun convert(from: BoundingBox): Double? {
+        if (from is CuboidBoundingBox)
+            return when (dimension) {
+                Dimension.LENGTH -> from.length
+                Dimension.WIDTH -> from.width
+                Dimension.HEIGHT -> from.height
+                Dimension.RADIUS -> null
+            }
+
+        if (from is Sphere)
+            return when (dimension) {
+                Dimension.RADIUS -> from.radius
+                else -> null
+            }
+
+        assert(false)
```

Comment:
Would it be better to raise an exception with a message here?
```suggestion
        throw IllegalArgumentException("Bounding box type '${from::class.name}' is not supported!")
```
Assertion does make sense, but this may provide better insight on what went wrong